### ISSUE TITLE
3.0.1 RE3 bugs report 

### ISSUE THREAD

**Issue by otamega72** [IntelOrca/biorand#338]
### Version

3.0.1

### Seed

Seed: R730-XAQF-ZTC885XZ3Z400NRPW490ZZZZZZZZZZZZZZZ

### Scenario

RE 3 (Jill)

### Player Character

Carlos

### What happened?

Seed: R730-XAQF-ZTC885XZ3Z400NRPW490ZZZZZZZZZZZZZZZ
1) crash : 
Connected 402:0 to 110:0 ( crashed if trigger press office nemesis before rpd front door nemesis)
Connected 104:1 to 205:2
Connected 412:2 [IRON PIPE, LIGHTER OIL, LIGHTER NO OIL] to 10E:0 (412->10e->back to 412 crash)
Connected 40F:1 [PARK KEY REAR] to 508:0 
 
2) random npc crashed the game . Game run smoothly if disable \mod_biorand\room
Connected 20C:0 (unblock) to 103:0
Connected 20C:15 [POWER CABLE, FUSE, OIL ADDITIVE, MACHINE OIL] to 215
Connected 218:1 to 302:1

open text file and Ctrl+F [ * ] to find the remark
[log_pl0.txt](https://github.com/IntelOrca/biorand/files/11144541/log_pl0.txt)

[re3 bug.zip](https://github.com/IntelOrca/biorand/files/11144561/re3.bug.zip)

3) npc problem : 
![Capture 6](https://user-images.githubusercontent.com/120957601/229684729-3e8085c2-e83a-4675-90ba-ae5f8b1fa8f2.JPG)

![Capture 5](https://user-images.githubusercontent.com/120957601/229684773-3767fcd6-5e07-431e-9eac-833a0d057e12.JPG)
![Capture 7](https://user-images.githubusercontent.com/120957601/229684814-6bb3f947-4ad7-4cc1-ade7-b06b23e6835c.JPG)

==============================
(not this seed)
4) Pc Kendo bug in warehouse 
![Capture](https://user-images.githubusercontent.com/120957601/229685435-74676ed3-2382-4171-876b-af2c7ff477e2.JPG)

**Comment by IntelOrca** [comment]
The nemesis crash is now fixed. I think it was a duplicate of #315.

**Comment by IntelOrca** [comment]
Which room did you get the NPC crash in?

**Comment by otamega72** [comment]
> 2\. random npc crashed the game . Game run smoothly if disable \mod_biorand\room
>     Connected 20C:0 (unblock) to 103:0
>     Connected 20C:15 [POWER CABLE, FUSE, OIL ADDITIVE, MACHINE OIL] to 215
>     Connected 218:1 to 302:1

103 , 215 , 302

**Comment by IntelOrca** [comment]
I have gone into all of those rooms from your 20C saved game, none of them crash. Did the crash happen when you entered the room or in the middle?

**Comment by otamega72** [comment]
1) It's weird , I regenerate this seed again. How come player character changed to Ark ? It used to be Carlos.
please change the player to Carlos before regenerating this seed

2) ( **c 20c 3rd item npc 07_2023-04-04 11^33.54.bio3** )
Connected 20C:15 [POWER CABLE, FUSE, OIL ADDITIVE, MACHINE OIL] 
_insert fuse , then crash. (both ark and carlos version)_

3) ( **c 20c npc 06_2023-04-02 19^37.43.bio3** )
( Connected 20C:0 (unblock) to 103:0 )
_Ark version no crash  , Carlos version crashed._

**Comment by IntelOrca** [comment]
![image](https://user-images.githubusercontent.com/1482259/236804984-5d34c206-971c-448f-8897-d2fef66c1ced.png)
I can't get it to crash, but the cutscene does get stuck. You can skip it however. This is probably the same issue was #278.

![image](https://user-images.githubusercontent.com/1482259/236805550-0df39551-3c02-4123-90cc-01273b6573cf.png)
I don't get a crash here either. Your crash might be related to the graphical issues, which could vary across game versions, settings and GPU.

**Comment by IntelOrca** [comment]
Okay looks like it is to do with our custom NPCs.

**Comment by IntelOrca** [comment]
Closing, main issue is now #352
